server/conf/application.conf

```diff
@@ -659,6 +659,9 @@ program_read_only_view_enabled = ${?PROGRAM_READ_ONLY_VIEW_ENABLED}
 program_eligibility_conditions_enabled = false
 program_eligibility_conditions_enabled = ${?PROGRAM_ELIGIBILITY_CONDITIONS_ENABLED}
 
+predicates_multiple_questions_enabled = false
+predicates_multiple_questions_enabled = ${?PREDICATES_MULTIPLE_QUESTIONS_ENABLED}
```

Comment:
I believe this needs to be in it's own PR just to protect against reverting this PR and removing this configuration after it's been set to true in other environment configurations.

Reply:
done